_data/sites.json

```diff
@@ -12478,6 +12500,16 @@
         ]
     },
 
+    {
+        "name": "Redfin",
+        "url": "https://www.redfin.com/myredfin/settings",
+        "difficulty": "easy",
+        "notes": "After logging in, click the Deactivate button at the bottom of the Settings page.",
```

Comment:
Do you know if this leads to deletion or just deactivation? i.e. can you reactivate it?

Reply:
It leads to the deletion of the account. I've just fixed the note.